go.mod

```diff
@@ -47,13 +48,18 @@ require (
 	github.com/containerd/containerd v1.5.7 // indirect
 	github.com/containerd/continuity v0.2.1 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/deepmap/oapi-codegen v1.10.1 // indirect
```

Comment:
So, this shouldn't get added here since nothing in the generated code relies on it, think we can revert most of the `go.mod` changes except for the stuff that pulls in `github.com/go-chi/chi/v5` in this commit. Everything else seems like it's just part of the global `go install` process which shouldn't actually affect this package's modules.